src/air-development/index.md

```diff
@@ -0,0 +1,9 @@
+# AIR Development
+
+At its core, a proof system can prove the validity of a statement $C(x,w)=y$ where $C$ is a representation of some logic, while $x$ and $w$ are inputs and $y$ the output of said logic. In the case of Stwo, that representation should be an Algebraic Intermediate Representation (AIR), which, very simply put, refers to a set of polynomials with constraints (i.e. equations) defined over them. A very simple example to illustrate this would be to think of three polynomials $f_1(x)$, $f_2(x)$, and $f_3(x)$ with a constraint that $f_1(x)$ multiplied by $f_2(x)$ equals $f_3(x)$.
```

Comment:
Usually the polynomials themselves do not satisfy these relations over the polynomial ring: if they are blinded they would not. Only their evaluations on a set domain would.